app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WelcomeController < ApplicationController
+  def index
+    @albums = Medium.albums.all.order(:rank).reverse.take(5)
```

Comment:
For numbers like 5 here, it's nice to define them as a constant like `TOP_MEDIA = 5` and then just pass in `TOP_MEDIA` for each function. That way, if you ever decide to show 10 items instead of 5, you only have to change one thing.